{
	"id": "oval:org.debian:def:278449202530558107977320036708219850430",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2013-1665 python-django",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 11",
			"product": "python-django"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2013-1665",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-1665"
			}
		],
		"description": "The XML libraries for Python 3.4, 3.3, 3.2, 3.1, 2.7, and 2.6, as used in OpenStack Keystone Essex and Folsom, Django, and possibly other products allow remote attackers to read arbitrary files via an XML external entity declaration in conjunction with an entity reference, aka an XML External Entity (XXE) attack.",
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:5464",
								"comment": "python-django DPKG is earlier than 1.4.4-1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 11 is installed"
			}
		]
	}
}
